### DISCUSSION TITLE
What does "force refresh" actually do?

### DISCUSSION THREAD

**Discussion by Stooovie** [fluidd-core/fluidd#1036] [Q&A] [answered]
I tend to look at Fluidd from my iOS devices, so I'm staring at the "force refresh" button every time I switch to the tab (or PWA). What does it actually do and can it be the default behavior at all times so I don't have to press the button to expedite the reconnection?

**Comment by pedrolamas** [comment, marked as answer]
Force Refresh is the same as using the browser Refresh/Reload button (F5 on the keyboard) as we show that when we lost connection to Moonraker, tried to connect again at least 3 times and failed to do so.

Having said that, browsers nowadays put tabs to sleep that cause the connection to close and I believe that is causing the Force Refresh screen to appear more usually than before - further investigation is needed to confirm, but I think this is the case.
